### PR TITLE
v2.0.5-unshaded release

### PR DESCRIPTION
**Pull request by sfc-gh-tzhang**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -779,9 +779,9 @@
         <configuration>
           <errorRemedy>failFast</errorRemedy>
           <!--
-                               The list of allowed licenses. If you see the build failing due to "There are some forbidden licenses used, please
-                              check your dependencies", verify the conditions of the license and add the reference to it here.
-                            -->
+                                         The list of allowed licenses. If you see the build failing due to "There are some forbidden licenses used, please
```
Comment (sfc-gh-lsembera) [inline pom.xml:782]: ?

File: src/test/java/net/snowflake/ingest/connection/UserAgentTest.java [hunk 2]
```diff
@@ -5,10 +5,12 @@
 import java.util.Properties;
 import net.snowflake.ingest.SimpleIngestManager;
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class UserAgentTest {
   @Test
+  @Ignore
```
Comment (sfc-gh-lsembera) [inline src/test/java/net/snowflake/ingest/connection/UserAgentTest.java:13]: Why aren't we changing also `RequestBuilder.DEFAULT_VERSION`?

Comment (sfc-gh-lsembera) [reply src/test/java/net/snowflake/ingest/connection/UserAgentTest.java:13]: Ah, we want the version in user agent without `-unshaded`, right?

File: pom.xml (hunk 1 above)
Comment (sfc-gh-tzhang) [reply pom.xml:782]: This is done by running the format.sh

File: src/test/java/net/snowflake/ingest/connection/UserAgentTest.java (hunk 2 above)
Comment (sfc-gh-tzhang) [reply src/test/java/net/snowflake/ingest/connection/UserAgentTest.java:13]: yup, at least that's what we did for the past few versions and we don't have an issue so far

**Comment by sfc-gh-xhuang** [comment]
I think we can skip 2.0.5-unshaded and release 2.0.6-unshaded

**Review by sfc-gh-lsembera — APPROVED** [review]

**Pull request closed without merging**